coralproject/talk
# [CORL-1167] Request Body Size

## What does this PR do?

Some requests may exceed size limitations and would need configuration to tweak, this exposes a new configuration parameter, `MAX_REQUEST_SIZE`, which is parsed by the [bytes](https://www.npmjs.com/package/bytes) package which will limit the request size of the body parser. If you experience errors related to a `PayloadTooLargeError`, then increasing this parameter from the default `500kb` could help.



## What changes to the GraphQL/Database Schema does this PR introduce?

None.



## How do I test this PR?



Send a body exceeding the defined value of `MAX_REQUEST_SIZE` and see an error. Increase the size to account for the increased body and see the error not appear after a server restart.
